### PR TITLE
REF: QRComponent

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

File: components/QRCodeComponent.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import Share from 'react-native-share';
+import loc from '../loc';
+import PropTypes from 'prop-types';
+
+const QRCodeComponent = ({
+  value,
+  isLogoRendered = true,
+  isMenuAvailable = true,
+  logoSize = 90,
+  size = (is.ipad() && 300) || 300,
```
Comment (limpbrains) [inline components/QRCodeComponent.js:16]: Is it always 300 ?

**Review by limpbrains — APPROVED** [review]

File: components/QRCodeComponent.js (hunk 1 above)
Comment (marcosrdz) [reply components/QRCodeComponent.js:16]: Default yes. If a different size is required then it gets passed as prop

Comment (limpbrains) [reply components/QRCodeComponent.js:16]: I mean, if default 300, then iPad checking can be removed

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**